lib/rules/this-in-template.js

```diff
@@ -116,7 +120,10 @@ module.exports = {
                     context.report({
                       node: reference.id,
                       loc: reference.id.loc,
-                      message: "Expected 'this'."
+                      message: "Expected 'this'.",
+                      fix(fixer) {
+                        return fixer.insertTextBefore(reference.id, 'this.')
+                      }
                     })
                   }
                 }
```

Comment:
I just chagne this-in-template.js and test/this-in-template, the rest just eslint autofix